react/components/Header/Warnings.js

```diff
@@ -5,8 +5,8 @@ const Info = () => {
   const twoOrders = true
   return (
     <Fragment>
-      <p><IconWarning color="#FFB100" /><strong className="pl3">Informações importantes:</strong></p>
-      <ul className="mv9 list ml0 pl0">
+      <p className="t-heading-6"><IconWarning color="#FFB100" /><strong className="pl3">Informações importantes:</strong></p>
```

Comment:
pra adicionar o token de cor no icone tambem, ao inves de hardcoded:

```suggestion
      <p className="t-heading-6"><span className="c-warning"><IconWarning color="currentColor" /></span><strong className="pl3">Informações importantes:</strong></p>
```

Reply:
só ve a formataçao dessa linha (espaços etc) antes de aceitar essa sugestao

Reply:
Feito!